### PR TITLE
add bookings to client profile page, minor styling on navbar, update …

### PR DESCRIPTION
**Pull request by zuccamia**
…routes

### CODE REVIEW

File: app/views/pages/caregiver_profile.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<div class="page-wrapper">
+  <div class="row">
+    <div class="left-column p-5">
+      <div class="user-photo">
+        <%= image_tag "https://images.unsplash.com/photo-1534286802268-99ddb388ed4c?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=934&q=80", class: "avatar-large", alt: "Profile Picture" %>
+        <%= link_to user_listings_path(@user), class: "btn btn-secondary" do %>
+          Check out All listings from <%= @user.first_name %>
```
Comment (worksofgenius) [inline app/views/pages/caregiver_profile.html.erb]: Can you please change this to: View all listings from <%= @user.first_name %>

File: app/views/shared/_my-booking-card.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<% if @my_bookings.empty? %>
+  <p>Wow, such empty! 👶</p>
```
Comment (worksofgenius) [inline app/views/shared/_my-booking-card.html.erb]: Lol! What if we add a call to action here, @zuccamia @attinore? Something like:
`<p>No bookings found.</p>`
`<p><%= link_to 'Create a new booking?', listings_path %></p>`

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -15,16 +15,15 @@
...
         <li class="nav-item dropdown">
-          <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
+          <%= image_tag "sakura-photo.jpg", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
           <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
-            <%= link_to "Action", "#", class: "dropdown-item" %>
-            <%= link_to "Another action", "#", class: "dropdown-item" %>
+            <%= link_to profile_path(current_user), class: "dropdown-item" do %>
+              👋 Welcome <%= current_user.first_name %>
+            <% end %>
+            <%= link_to "Bookings", "#", class: "dropdown-item" %>
```
Comment (attinore) [inline app/views/shared/_navbar.html.erb]: Wondering should make the flow for checking all the bookings 
- from the user profile OR
- bookings 
- or both? (in this case, the user profile would only contain summarized info, and bookings will have all bookings made with more detailed info including notes, which a page we don't have yet)
? 
@zuccamia @worksofgenius

File: app/views/shared/_received-booking-card.html.erb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<% @my_listings.each do |listing| %>
+  <% if listing.bookings.nil? %>
+    <p>Wow, such empty! 👶</p>
```
Comment (attinore) [inline app/views/shared/_received-booking-card.html.erb]: nice to add this user friendly message, maybe rephrase this sentence?

File: app/views/shared/_received-booking-card.html.erb [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  <% else %>
+  <% listing.bookings.each do |booking| %>
+    <div class="booking-list-card">
+      <%= link_to listing_path(listing) do %>
```
Comment (attinore) [inline app/views/shared/_received-booking-card.html.erb]: This link probably need to go to a individual booking detail page or show a popup. Let me know your thoughts.

File: app/views/shared/_navbar.html.erb (hunk 3 above)
Comment (worksofgenius) [inline app/views/shared/_navbar.html.erb]: Should this be: 
`<%= link_to "Bookings", bookings_path, class: "dropdown-item" %>` ?

File: app/views/shared/_received-booking-card.html.erb (hunk 4 above)
Comment (worksofgenius) [inline app/views/shared/_received-booking-card.html.erb]: Same as above. What if we add a call to action here? Something like:
<p>No bookings found.</p>
<p><%= link_to 'Create a new booking?', listings_path %></p>

File: app/views/shared/_user-details.html.erb [hunk 6]
```diff
@@ -0,0 +1,14 @@
+<h2><%= current_user.first_name %> <%= current_user.last_name %></h2>
+<p><%= current_user.age_range %>, <i class="fas fa-venus"></i></p>
```
Comment (worksofgenius) [inline app/views/shared/_user-details.html.erb]: The gender icon was a placeholder for the pitch. Either we have to connect it to the user profile or delete it. (I'm ok deleting it.)

**Review by worksofgenius — COMMENTED** [review]
Please read through my comments.

File: app/views/shared/_received-booking-card.html.erb (hunk 5 above)
Comment (worksofgenius) [reply app/views/shared/_received-booking-card.html.erb]: I think going to an individual booking detail page is better than a popup, especially for mobile browsers.

File: app/views/shared/_navbar.html.erb (hunk 3 above)
Comment (worksofgenius) [reply app/views/shared/_navbar.html.erb]: Checking the user profile seems like the easiest flow. Booking.com and other travel sites seem to use it.

File: app/views/pages/caregiver_profile.html.erb (hunk 1 above)
Comment (zuccamia) [reply app/views/pages/caregiver_profile.html.erb]: Yes! I will include this in my next commit.

File: app/views/shared/_my-booking-card.html.erb (hunk 2 above)
Comment (zuccamia) [reply app/views/shared/_my-booking-card.html.erb]: @worksofgenius Yeah I think a call to action would work well here too! Will add this! 
Just a suggestion but what 'Find a caregiver' for 'Create a new booking?'? (I thought it sounds a bit more intuitive to me 🤔 )

Comment (worksofgenius) [reply app/views/shared/_my-booking-card.html.erb]: I like it!

File: app/views/shared/_navbar.html.erb (hunk 3 above)
Comment (zuccamia) [reply app/views/shared/_navbar.html.erb]: @attinore I agree with @worksofgenius. I was thinking we can use something like [this](https://jqueryui.com/tooltip/) for the notes and `accept`/`cancel` with this type of js.erb file like [this tutorial](https://flatironschool.com/blog/jquery-and-javascript-meet-erb-embedded-ruby)

File: app/views/shared/_received-booking-card.html.erb (hunk 5 above)
Comment (zuccamia) [reply app/views/shared/_received-booking-card.html.erb]: @worksofgenius @attinore if we are making the flow so that the user can check all their bookings on their profile page (kind of like a dashboard), I think we probably won't need an individual booking detail page?

File: app/views/shared/_navbar.html.erb (hunk 3 above)
Comment (zuccamia) [reply app/views/shared/_navbar.html.erb]: @worksofgenius yes but same as above, I was thinking whether to let the user check all their bookings on their profile page or go to a separate booking details page, so I temporarily put it as `#` 😅

File: app/views/shared/_user-details.html.erb (hunk 6 above)
Comment (zuccamia) [reply app/views/shared/_user-details.html.erb]: @worksofgenius That depends on whether we want our user to have a gender attribute or not. If it's a necessity for an user, it's not too complicated to add. What do you and @attinore think?

File: app/views/shared/_received-booking-card.html.erb (hunk 4 above)
Comment (zuccamia) [reply app/views/shared/_received-booking-card.html.erb]: @worksofgenius My understanding was that an action here will be to create more received bookings, which I couldn't think of much more than to either 'Create more listings' (similar to what you suggested I think) or improve user profile to get more traction 😄

File: app/views/shared/_user-details.html.erb (hunk 6 above)
Comment (worksofgenius) [reply app/views/shared/_user-details.html.erb]: @zuccamia At this point, I think skills are more important than gender.

File: app/views/shared/_received-booking-card.html.erb (hunk 4 above)
Comment (zuccamia) [reply app/views/shared/_received-booking-card.html.erb]: Yes, we can add "You have no bookings"
